install/src/command.rs

```diff
@@ -1283,3 +1283,30 @@ pub fn run(
         }
     }
 }
+
+pub fn list(config_file: &str) -> Result<(), String> {
+    let config = Config::load(config_file)?;
+
+    let entries = fs::read_dir(&config.releases_dir).map_err(|err| format!("Failed to read install directory, double check that your configuration file is correct: {err}"))?;
```

Comment:
Hmm, so pasting the function in and running `cargo fmt` myself doesn't seem to have any effect. Not sure why, but in any case, this line goes out to 175 characters. Can you please make the following modification manually.
```suggestion
    let entries = fs::read_dir(&config.releases_dir).map_err(|err| {
        format!(
            "Failed to read install directory, \
            double check that your configuration file is correct: {err}"
        )
    })?;
```

Reply:
I am on Linux using LF, and indenting using a tab size of 4. 

Or is there a fmt tool I am missing? 

perhaps adding a `.editorconfig` rc file would help?

Reply:
No, I don't think you're missing anything on config side. As mentioned, I also ran `cargo fmt` on my system (Mac) with your branch and it didn't modify the line either.

Not sure why it didn't take, but thank you for the manual fix!